kratos/includes/property_accessor.h

```diff
@@ -0,0 +1,244 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                     Kratos default license: kratos/license.txt
+//
+//  Main authors:    Alejandro Cornejo
+//                   Riccardo Rossi
+//
+//
+
+
+#if !defined(KRATOS_PROPERTY_ACCESSOR_H_INCLUDED)
+#define KRATOS_PROPERTY_ACCESSOR_H_INCLUDED
+
+// System includes
+#include "includes/properties.h"
+#include "geometries/geometry.h"
+
+// External includes
+
+// Project includes
+
+namespace Kratos
+{
+
+///@name Kratos Globals
+///@{
+
+///@}
+///@name Type Definitions
+///@{
+
+///@}
+///@name  Enum's
+///@{
+
+///@}
+///@name  Functions
+///@{
+
+///@}
+///@name Kratos Classes
+///@{
+
+/**
+ * PropertyAccessor sets a proper way of returning a value from the properties
+ * It is used to properly interpolate material properties according to temperature, tables, etc...
+*/
+class KRATOS_API(KRATOS_CORE) PropertyAccessor
```

Comment:
Move the code to  a cpp. There is no template, so it's fine

Reply:
It will be a template for the variable type I think

Reply:
Huuum, remove KRATOS_API then